Wikipedia/UI-V5/WMFTableOfContentsCell.swift

```diff
@@ -0,0 +1,167 @@
+
+import UIKit
+
+// MARK: - Cell
+public class WMFTableOfContentsCell: UITableViewCell {
+
+    @IBOutlet var sectionSelectionBackground: UIView!
+    @IBOutlet var sectionTitle: UILabel!
+    @IBOutlet var selectedSectionIndicator: UIView!
+    @IBOutlet var indentationConstraint: NSLayoutConstraint!
+    @IBOutlet var leadSectionBorder: UIView!
+    @IBOutlet var sectionBorder: UIView!
+    
+    // MARK: - Init
+    public required init?(coder aDecoder: NSCoder) {
+        section = nil
+        super.init(coder: aDecoder);
+        self.indentationWidth = 10
+    }
+    
+    // MARK: - UIView
+    public override func awakeFromNib() {
+        super.awakeFromNib()
+        self.selectedSectionIndicator.alpha = 0.0
+        self.sectionSelectionBackground.backgroundColor = UIColor.wmf_tableOfContentsSelectionBackgroundColor()
+        self.sectionSelectionBackground.alpha = 0.0
+        self.selectionStyle = .None
+    }
+    
+    // MARK: - Accessors
+    public var section: MWKSection?{
+        didSet(section) {
+            self.sectionTitle.text = WMFTableOfContentsCell.titleText(self.section)
```

Comment:
Also I considered just making them functions outside of the implementation of the class…